docs/source/guides/speed.rst

```diff
@@ -283,32 +318,35 @@ Check validation every n epochs
 
 **Use when:** You have a small dataset, and want to run less validation checks.
 
-You can limit validation check to only run every n epochs using the `check_val_every_n_epoch` Trainer flag.
+You can limit validation check to only run every n epochs using the ``check_val_every_n_epoch`` Trainer flag.
 
 .. testcode::
 
-    # DEFAULT
+    # default
     trainer = Trainer(check_val_every_n_epoch=1)
 
+    # runs validation after every 7th Epoch
+    trainer = Trainer(check_val_every_n_epoch=7)
 
-Set validation check frequency within 1 training epoch
-======================================================
+
+Validation within Training Epoch
+================================
 
 **Use when:** You have a large training dataset, and want to run mid-epoch validation checks.
 
 For large datasets, it's often desirable to check validation multiple times within a training loop.
```

Comment:
within a training loop or epoch? this line and the next one are confusing
I also don't really see how this is related to speed

Reply:
epoch.. will update.